sfcasts/ep1/tdd.md

```diff
@@ -1,43 +1,55 @@
 # TDD
 
-All right. So one of the problems is that when Bob the park ranger sees the
+All right. So one of the problems is that when Bob, our park ranger, sees the
 dinosaur size... he can't remember if these are in meters... or centimeters...
 which makes a big difference when you step into a cage.
 
 A better way might be to just use words like small, medium, or large. So... let's
 do that!
 
+## What is TDD?
+
 *But*, to add this feature, we're going to use a philosophy called
-test driven development or TDD. TDD is basically a buzzword that describes a
+Test Driven Development or TDD. TDD is basically a buzzword that describes a
 4-step process for writing your test *before* the feature.
 
-Step 1: write a test for the feature. Step 2: run your test and watch it fail...
-since you haven't created that feature yet! Step 3: Write as little code as possible
-to get your test to pass. And Step 4: now that it's passing, refactor your code
-if needed to make it more awesome.
+Step 1: Write a test for the feature. Step 2: Run your test and watch it fail...
```

Comment:
This sentence (or the previous one) needs to be altered because they have almost the same text (`test before the feature` then `write a test for the feature`). When reading these, the repetition stuck out.